### PR TITLE
NO-ISSUE: Change GCS bucket name in prow jobs scraper

### PR DESCRIPTION
**Pull request by danmanor**
Change GCS bucket name in prow jobs scraper

### CODE REVIEW

File: src/prowjobsscraper/equinix_metadata.py [hunk 1]
```diff
@@ -47,6 +48,11 @@ def _set_equinix_metadata(self, job: ProwJob) -> None:
         except exceptions.ClientError as e:
             logger.debug("Equinix metadata are missing from %s: %s", metadata_path, e)
 
+        except auth_exceptions.InvalidOperation as e:
```
Comment (osherdp) [inline src/prowjobsscraper/equinix_metadata.py:51]: before patching it let's first identify what URI we're unable to access
maybe even at first stage do:
```
except auth_exceptions.InvalidOperation as e:
    raise RuntimeError(f"Forbidden access to {metadata_path}") from e
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: openshift/template.yaml [hunk 2]
```diff
@@ -433,6 +433,8 @@ parameters:
   value: "300m"
 - name: PROW_JOBS_SCRAPER_MEMORY_REQUEST
   value: "400Mi"
+- name: GCS_BUCKET_NAME
```
Comment (osherdp) [inline openshift/template.yaml:436]: I think you're missing a usage of that parameter

**Review by osherdp — CHANGES_REQUESTED** [review]

File: openshift/template.yaml (hunk 2 above)
Comment (danmanor) [reply openshift/template.yaml:436]: Sorry my bad

**Pull request merged**